fabrics.c

```diff
@@ -1044,10 +1041,12 @@ int build_options(char *argstr, int max_len, bool discover)
 	    (strncmp(fabrics_cfg.transport, "loop", 4) &&
 	     add_int_argument(&argstr, &max_len, "ctrl_loss_tmo",
 			      fabrics_cfg.ctrl_loss_tmo, true)) ||
+	    (fabrics_cfg.fast_io_fail_tmo != -1 &&
```

Comment:
As far I understand the kernel code, fast_io_fail_tmo = -1 is used to disable this feature. Which means it -1 is a valid value.

Reply:
It was filtered out by add_int_argument() before my patch, so I did the same to avoid introducing regressions

Reply:
Got it, but I think we should figure out how it supposed to work correctly.

Reply:
> Got it, but I think we should figure out how it supposed to work correctly.

I will investigate and eventually submit another pull request